### ISSUE TITLE
Add label "Congratulations, all pictures in this album have been either uploaded or marked as not for upload."

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#5619]
In the custom picker, with "Show already actioned pictures" disabled, this is how the screen looks after all pictures have been either uploaded or marked as not for upload:

![Screenshot_20240310-162611.png](https://github.com/commons-app/apps-android-commons/assets/99590/8166171f-bdd7-436f-9be7-599fd5c43642)

Ideally, it should instead display a message such as:

"Congratulations, all pictures in this album have been either uploaded or marked as not for upload."

**Comment by neeldoshii** [comment]
Can we either display "An vector" like this?

If no image is left to upload or masked as not for upload, we can display svg image & display text like 
1. Custom Picker finalised: Every image is now either uploaded or marked.
2. Album perfection achieved: Every photo uploaded or sorted.
I don't think congratulations is right word as if some person has marked all images not for upload will get text congratulation might be incorrect from UX copywriting pov.

![image](https://github.com/commons-app/apps-android-commons/assets/60827173/5954095c-0dde-492e-92b7-365f6b8fafae)
[Credits](https://bootcamp.uxdesign.cc/designing-a-food-delivery-app-from-scratch-a-ux-case-study-82fd100f4e6a)

What do you think about it @nicolas-raoul. I can tackle this while my wikimedia id is banned.

**Comment by nicolas-raoul** [comment]
@neeldoshii Sorry for the delay! Yes vetor image would be nice, if possible.

"Congratulations": Even marking all pictures as not for upload is good, so I think it is a good opportunity to make people feel good. :-) Maybe something like "Good job! [...]"

Maybe it could even suggest the user to perform peer reviews or go take nearby pictures, with buttons to launch these activities.

**Comment by starrain1031** [comment]
I would like to try to solve this problem. Would you like to assign it to me?

**Comment by Thejas775** [comment]
> I would like to try to solve this problem. Would you like to assign it to me?

Are you still working on this issue ?

**Comment by starrain1031** [comment]
> > I would like to try to solve this problem. Would you like to assign it to me?
> 
> Are you still working on this issue ?

Yes. However, based on my limited skills, it might take me sometime.

**Comment by nicolas-raoul** [comment]
@Thejas775 Thanks for your enthusiasm, however better work on unassigned bugs (the app has enough bugs for everybody haha) next time.

@starrain1031 Sorry for unusual workflow this time. Maybe review Thejas775's pull request, improve it if possible, then work on a second bug if you want? :-)

**Comment by Thejas775** [comment]
I'm really sorry but this was my first contribution to open source as well. This was a good first issue that I thought I could solve.

**Comment by nicolas-raoul** [comment]
@Thejas775 That's understandable :-) You might want to read https://github.com/commons-app/commons-app-documentation/blob/master/android/Volunteers-welcome!.md#developers which has the link to unassigned bugs. I hope you will enjoy fixing other bugs :-)

**Comment by Thejas775** [comment]
> @Thejas775 Thanks for your enthusiasm, however better work on unassigned bugs (the app has enough bugs for everybody haha) next time.
> 
> @starrain1031 Sorry for unusual workflow this time. Maybe review Thejas775's pull request, improve it if possible, then work on a second bug if you want? :-)

Will follow the workflow from next time.

**Comment by Thejas775** [comment]
> @Thejas775 That's understandable :-) You might want to read https://github.com/commons-app/commons-app-documentation/blob/master/android/Volunteers-welcome!.md#developers which has the link to unassigned bugs. I hope you will enjoy fixing other bugs :-)

Yeah sure seems intresting.

**Comment by nicolas-raoul** [comment]
@starrain1031 Any progress on this? :-)

**Comment by AlexD4110** [comment]
> @starrain1031 Any progress on this? :-)

If @starrain1031 is not working on this. I can do this.

**Comment by nicolas-raoul** [comment]
@AlexD4110 Thanks for your enthusiasm! I just assigned you to the other issue, when done please ask again here. Thanks a lot! :-)

**Comment by yuvraj-coder1** [comment]
Hi, can i work on this issue @nicolas-raoul
